app/settings/surveys/attribute-editor.html

```diff
@@ -92,19 +92,30 @@
            <label class="tgl-btn" for="switchprivateresponse"></label>
         </div>
        </div>
-       <div class="form-field switch" ng-if="canDisplay()">
-            <label translate="app.default_value">Default value (optional):</label>
-            <div class="form-field">
-              <div ng-switch="editAttribute.input">
-                  <input ng-switch-when="location" type="text" placeholder="{{ 'form.default_location_placeholder'|translate }}" ng-model="editAttribute.default">
-                  <input ng-switch-when="date" type="text" date-time="editAttribute.default" ng-model="editAttribute.default">
-                  <input ng-switch-when="int" type="number" step="1" ng-model="editAttribute.default">
-                  <input ng-switch-when="decimal" type="number" ng-model="editAttribute.default">
-                  <input ng-switch-default type="text" placeholder="{{ 'form.default_default_placeholder'|translate }}" ng-model="editAttribute.default">
-              </div>
+       <div class="form-field switch" ng-if="canDisplayDefaultValue()">
+          <label ng-if="canDisplayDefaultValue()" translate="app.default_value">Default value (optional):</label>
+          <div id="form-field" class="form-field">
```

Comment:
Could you make sure to connect the label to the input-field here with `for` and `id`?

Reply:
Alright. Will do so.